ports/renesas-ra/boards/RA4M1_CLICKER/ra/board/ra4m1_clicker/board.h

```diff
@@ -0,0 +1,65 @@
+/***********************************************************************************************************************
+ * Copyright [2015] Renesas Electronics Corporation and/or its licensors. All Rights Reserved.
+ *
+ * This file is part of Renesas RA Flexible Software Package (FSP)
+ *
+ * The contents of this file (the "contents") are proprietary and confidential to Renesas Electronics Corporation
+ * and/or its licensors ("Renesas") and subject to statutory and contractual protections.
+ *
+ * This file is subject to a Renesas FSP license agreement. Unless otherwise agreed in an FSP license agreement with
+ * Renesas: 1) you may not use, copy, modify, distribute, display, or perform the contents; 2) you may not use any name
```

Comment:
This license does not look very friendly to open source.  Having this file in this repository seems to contradict this clause (1), that the file may not be used/copied/etc.

Reply:
I understand.  I have removed board/*/ra and linke to lib/fsp files.